### ISSUE TITLE
[mac] fork 1.0.78.1 cli not working on high sierra 10.13.6

### ISSUE THREAD

**Issue by khoan** [ForkIssues/Tracker#611]
```
$ sw_vers 
ProductName:	Mac OS X
ProductVersion:	10.13.6
BuildVersion:	17G6030
$ fork
dyld: Library not loaded: @rpath/libswiftAppKit.dylib
  Referenced from: /usr/local/bin/fork
  Reason: image not found
Abort trap: 6
```

fork cli version 1.0.73.5 works without any errors.

**Comment by DanPristupov** [comment]
Swift 5 command line programs require the Swift 5 libraries. These libraries are included in the OS starting with macOS Mojave 10.14.4. When running on earlier versions of macOS you must install the following package manually.
https://support.apple.com/kb/DL1998?locale=en_US

**Comment by phuongnd08** [comment]
Thank for the solution. It should be nice if this to be automatically done when running `brew cask install fork`
